packages/block-library/src/post-featured-image/style.scss

```diff
@@ -12,6 +12,11 @@
 		box-sizing: border-box;
 	}
 
+	:where(img),
+	:where(.wp-block-post-featured-image__overlay) {
+		border-style: solid;
```

Comment:
Could you explain why we need to apply a solid border style by default? With this change, I'm seeing a solid border on every featured image:

![image](https://user-images.githubusercontent.com/1645628/191490255-f2936967-9665-4411-86dd-a62c3cc3c053.png)

![image](https://user-images.githubusercontent.com/1645628/191490480-86225930-2820-44dd-8380-4681a41c22d0.png)

![image](https://user-images.githubusercontent.com/1645628/191490751-406b4e1a-abc2-414a-ab1f-4d4bbf964a6b.png)

Reply:
Apologies, I missed this as the image I was testing with had black edges. 

It was added because if you only set a border color and width via theme.json then no border style is set as core adds `img { border: none }`. Setting the border style via theme.json then introduces other edge cases when the block doesn't support border style, as is the case with our image-related blocks at the moment. 

I've removed (https://github.com/WordPress/gutenberg/pull/44286/commits/b0eeade929546472aabcbecbf4ba38466658b0bd) these fallbacks and the decision to omit border style support from the post featured image block can be revisited in a follow-up.

Reply:
Ahh right, thanks for explaining. This makes sense and sounds like a good follow-up.